scripts/version-linter.py

```diff
@@ -250,18 +269,18 @@ def assert_commit(commit, desc):
 
   subprocess.run(['git','fetch'],capture_output=False)
 
-  base_branch_commit=branch_commit(base_branch)
-  download_type_shapes('base',base_branch,base_branch_commit)
+  base_branch_commit = find_latest_type_shape_ref_on(base_branch,n=10)
+  download_type_shape('base',base_branch,base_branch_commit)
 
   print('')
 
-  release_branch_commit=branch_commit(release_branch)
-  download_type_shapes('release',release_branch,release_branch_commit)
+  release_branch_commit=find_latest_type_shape_ref_on(release_branch, n=10)
```

Comment:
As I mentioned in `develop` twin of this PR, I'm not sure this 10-commit check is a good idea, or necessary.